### ISSUE TITLE
[1.8.1] For of loops still being transpiled in modern browser bundle

### ISSUE THREAD

**Issue by jeffm24** [meteor/meteor#10622]
I tried searching through the issues to see if anyone mentioned this, but couldn't find anything. I've noticed that in modern browsers (Chrome 75, Chromium Edge 77, Firefox 68), the code is pretty much untouched by the transpiler as intended, but for of loops are still transpiled for some reason. 

So for example:
```javascript
for (const doc of docs) {
    ...
}
```
becomes something like:
```javascript
for (var _iterator = docs, _isArray = Array.isArray(_iterator), _i = 0, _iterator = _isArray ? _iterator : _iterator[Symbol.iterator]();;) {
    var _ref;

    if (_isArray) {
        if (_i >= _iterator.length) break;
        _ref = _iterator[_i++];
    } else {
        _i = _iterator.next();
        if (_i.done) break;
        _ref = _i.value;
    }

    ...
}
```
I'm not sure if this is intended or not, but I don't really see why it should be happening.

**Comment by benjamn** [comment]
Good catch! I definitely think we can skip this transform for the modern and server bundles, and adjust the [minimum browser versions](https://github.com/meteor/meteor/blob/release-1.8.2/packages/modern-browsers/modern.js) accordingly.

**Comment by benjamn** [comment]
Closing since this issue will be fixed in Meteor 1.8.2 by the commits linked above.
